<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="col_left">
      <item>
       <widget class="QPushButton" name="folder_btn">
        <property name="text">
         <string>Відкрити папку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="image_list">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="col_right">
      <item>
       <layout class="QHBoxLayout" name="btn_row">
        <item>
         <widget class="QPushButton" name="del_btn">
          <property name="text">
           <string>Видалити</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/trash.svg</normaloff>../../../Downloads/trash.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_btn">
          <property name="text">
           <string>Зберегти</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/device-floppy.svg</normaloff>../../../Downloads/device-floppy.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_btn">
          <property name="text">
           <string>Скасувати</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/backspace.svg</normaloff>../../../Downloads/backspace.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forward">
          <property name="text">
           <string>Повернути</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="rotate_left">
          <property name="text">
           <string>Поворот</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/u-turn-left.svg</normaloff>../../../Downloads/u-turn-left.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="current_image">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>500</height>
         </size>
        </property>
        <property name="text">
         <string>Картинка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_file"/>
    <addaction name="save"/>
    <addaction name="open_folder"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Реадагування</string>
    </property>
    <addaction name="cancel"/>
    <addaction name="reset"/>
   </widget>
   <widget class="QMenu" name="menu_effects">
    <property name="title">
     <string>Ефекти</string>
    </property>
    <addaction name="black_white"/>
    <addaction name="sharpen"/>
    <addaction name="blur"/>
   </widget>
   <widget class="QMenu" name="menu_correction">
    <property name="title">
     <string>Корекція</string>
    </property>
    <addaction name="contrast"/>
    <addaction name="brightness"/>
    <addaction name="saturation"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_effects"/>
   <addaction name="menu_correction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="open_file">
   <property name="text">
    <string>Відкрити файл</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Зберегти</string>
   </property>
  </action>
  <action name="cancel">
   <property name="text">
    <string>Скасувати</string>
   </property>
  </action>
  <action name="reset">
   <property name="text">
    <string>Скасувати всі зміни</string>
   </property>
  </action>
  <action name="black_white">
   <property name="text">
    <string>Чорно-білий</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Контраст</string>
   </property>
  </action>
  <action name="sharpen">
   <property name="text">
    <string>Різкість</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Яскравість</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Насиченість</string>
   </property>
  </action>
  <action name="blur">
   <property name="text">
    <string>Розмиття</string>
   </property>
  </action>
  <action name="contrast">
   <property name="text">
    <string>Контрастність</string>
   </property>
  </action>
  <action name="brightness">
   <property name="text">
    <string>Яскравість</string>
   </property>
  </action>
  <action name="saturation">
   <property name="text">
    <string>Насичненсть</string>
   </property>
  </action>
  <action name="open_folder">
   <property name="text">
    <string>Відкрити папку</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
